esi_leap/api/controllers/v1/lease.py

```diff
@@ -207,15 +205,16 @@ def _lease_get_all_authorize_filters(cdict,
         }
 
         if view == 'all':
```

Comment:
What is the 'view' parameter?

Reply:
Oh, that's a CLI parameter that a user using the CLI can set. By default the CLI will return only live offers and leases. The '-view all' option tells the controller to send *all* offers/leases instead (possibly for generating a report).

Reply:
Got it.